pvc_to_pvs_image_import/_topics/preparing_powervc_node.md

```diff
@@ -0,0 +1,223 @@
+### Assumptions:
+
+1. You have the latest installation of the ManageIQ base `>= 98cb43d08130ae...`.
+
+
+2. You have `ansible-runner` application installed in your operating system.
+
+   
+3. Your ManageIQ base is located under `<MIQ_PATH>`.
+
+### Cloning IBM Cloud Plugin Repository (needed only for the ALPHA release)
+
+1. Enter the plugin directory `cd <MIQ_PATH>/plugins`.
+
+
+2. Clone IBM Cloud Plugin repository: `git clone git@github.com:iv1111/manageiq-providers-ibm_cloud.git`.
+
+
+3. Checkout the image-import tag: `git checkout image_import_alpha`.
+
+
+4. Enter the "bundle.d" directory: `cd <MIQ_PATH>/bundle.d`.
+
+
+5. Override IBM Cloud Plugin: `echo "ensure_gem 'manageiq-providers-ibm_cloud', :path => '<MIQ_PATH>/manageiq/plugins/manageiq-providers-ibm_cloud'" > overrides.rb`.
+
+
+6. Turn off your rails server if it's running.
+
+
+7. Enter the plugin directory: `cd <MIQ_PATH>`.
+
+
+8. Execute `bundle install`.
+
+
+9. Start your rails server.
+
+
+### Preparing IBM PowerVC Node:
+
+1.  Using package manager install `python3.6`, `pip3`, `virtualenv` and `powervc-image` applications 
+    
+    as well as `libselinux-python3` library.
+
+
+2.  Create sessions directory `/home/sessions` with enough storage to hold GBs of transient images at once.
+
+
+3.  Create a virtual Python environment: `virtualenv --system-site-packages /home/sessions/venv`.
+
+
+4.  Activate the newly created python environment:  `source /home/sessions/venv/bin/activate`.
+
+
+5.  Install Python packages inside the Virtual Environment using: `pip3 install ntpath base64 pathlib pycryptodome ibm_boto3`.
+
+
+6.  Exit the virtual environment by running: `deactivate`.    
+
+ 
+7.  Place valid PowerVC resource file under '`/opt/ibm/powervc/powervcrc`' with similar content:
+    
+    `export OS_IDENTITY_API_VERSION=3`
+    
+    `export OS_AUTH_URL=https://host:5000/v3/`
+    
+    `export OS_CACERT=/etc/pki/tls/certs/powervc.crt`
+    
+    `export OS_REGION_NAME=RegionOne`
+    
+    `export OS_PROJECT_DOMAIN_NAME=Default`
+    
+    `export OS_PROJECT_NAME=ibm-default`
+    
+    `export OS_TENANT_NAME=$OS_PROJECT_NAME`
+    
+    `export OS_USER_DOMAIN_NAME=Default`
+    
+    `export OS_USERNAME=root`
+    
+    `export OS_PASSWORD=password`
+    
+    `export OS_COMPUTE_API_VERSION=2.46`
+    
+    `export OS_NETWORK_API_VERSION=2.0`
+    
+    `export OS_IMAGE_API_VERSION=2`
+    
+    `export OS_VOLUME_API_VERSION=2`
+
+
+8. Before proceeding make sure to test image exporting:
+
+   * Execute `source /opt/ibm/powervc/powervcrc`.
+     
+   * Note the name of the image you want to export in the output: `powervc-image list`
+   
+   * Export the image `powervc-image export -i <image_name> -p /home/sessions/image.ova`
+
+
+### Add a PowerVS instance
```

Comment:
You could provide a link to the existing PowerVS provider documention:
- https://www.manageiq.org/docs/reference/latest/managing_providers/index.html#ibm-power-systems-virtual-servers-providers
- https://github.com/ManageIQ/manageiq-documentation/blame/master/managing_providers/_topics/adding_ibm_cloud_powervs_providers.md